src/grafana_dashboards/grafana-agent-k8s_rev1.json.tmpl

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
The following metrics seem worthwhile for an alert rule.
```
# HELP prometheus_target_interval_length_seconds Actual intervals between scrapes.
```

The following metrics seem worthwhile for a panel.
```
# HELP agent_request_duration_seconds Time (in seconds) spent serving HTTP requests.
# HELP agent_wal_samples_appended_total Total number of samples appended to the WAL
# HELP agent_wal_storage_active_series Current number of active series being tracked by the WAL storage
# HELP process_cpu_seconds_total Total user and system CPU time spent in seconds.
# HELP process_resident_memory_bytes Resident memory size in bytes.
# HELP process_virtual_memory_bytes Virtual memory size in bytes.
# HELP prometheus_sd_discovered_targets Current number of discovered targets.
# HELP promhttp_metric_handler_requests_in_flight Current number of scrapes being served.
# HELP promhttp_metric_handler_requests_total Total number of scrapes by HTTP status code.
# HELP prometheus_target_scrapes_exceeded_sample_limit_total Total number of scrapes that hit the sample limit and were rejected.
# HELP prometheus_target_scrapes_sample_duplicate_timestamp_total Total number of samples rejected due to duplicate timestamps but different values.
# HELP prometheus_target_scrapes_sample_out_of_bounds_total Total number of samples rejected due to timestamp falling outside of the time bounds.
# HELP prometheus_target_scrapes_sample_out_of_order_total Total number of samples rejected due to not being out of the expected order.
```